examples/benchmarks/dist_inference_cpp.py

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) Microsoft Corporation.
```

Comment:
After merging, we don't need it any more

Reply:
Merged.